### PR TITLE
Add types

### LINKED ISSUE #39: Export types
Now that [Agenda is TypeScript](https://github.com/agenda/agenda/pull/1129), we should export types directly from `human-interval`.

:wave: @paulmelnikow because you've written Types for this previously at https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/human-interval ;-)

### PR DESCRIPTION
**Pull request by simison**
Resolves #39 

Copied over from https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/human-interval

Using this just to practice and get better TS understanding. 😁

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+declare function humanInterval(time?: string): number | undefined;
+
+declare namespace humanInterval {
```
Comment (simllll) [inline index.d.ts]: not quite sure what this namespace definition is for? is the languagemap something this library exports? and should the types of it be overwriteable?

the only export this library has according to documentation is the function "humanInterval(..)" as default export..

**Review by simllll — APPROVED** [review]
not quite sure about the namepsace, but otherwiese lgtm

File: index.d.ts (hunk 1 above)
Comment (simison) [reply index.d.ts]: Right, that used to be a thing but I forgot it got already refactored out, here:

https://github.com/agenda/human-interval/pull/37/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L66

Comment (simison) [reply index.d.ts]: Removed in https://github.com/agenda/human-interval/pull/43/commits/ed1c53691274682e96acae2b4b9434106b37e1c2 — Thanks for pointing it out!

**Review by simllll — APPROVED** [review]

**Pull request merged**